ubuntu_mint.sh

```diff
@@ -1,152 +1,138 @@
-    elif [ ${version} == '3' ] ; then
-        source ./travis-run-tests.sh python3
-    fi
-}
-
-
-set -euo pipefail
-IFS=$'\n\t'
+source utils.sh
 
 # system update
-eval sudo apt-get -y update
-eval sudo apt-get install -y ${apt_package_list}
-export PATH="${HOME}/.local/bin:${PATH}"
-eval python -m pip install --user --upgrade pip
-eval pip install --user ${pip_package_list}
+sudo apt-get -y update
+sudo apt-get install -y ${apt_package_list}
+python -m pip install --user --upgrade pip
+pip install --user ${pip_package_list}
 
 install_dir=${HOME}/opt
 mkdir -p ${install_dir}
 
 # need to put libhdf5.so on LD_LIBRARY_PATH
-export LD_LIBRARY_PATH=${hdf5_libdir}
-export LIBRARY_PATH=${hdf5_libdir}
 echo "export LD_LIBRARY_PATH=${hdf5_libdir}" >> ~/.bashrc
-
-build_moab
-
-build_dagmc
-
-install_pyne $1
-
+echo "export LIBRARY_PATH=${hdf5_libdir}" >> ~/.bashrc
+
+source ~/.bashrc
+
+############
+### MOAB ###
+############
+
+# pre-setup
+cd ${install_dir}
+check_repo moab
+mkdir -p moab
+cd moab
+
+# clone and version
+git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab-repo
+cd moab-repo
+mkdir -p build
+cd build
+
+# cmake, build and install
+cmake ../ -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
+            -DBUILD_SHARED_LIBS=ON \
+            -DENABLE_PYMOAB=ON \
+            -DENABLE_BLASLAPACK=OFF \
+            -DENABLE_FORTRAN=OFF \
+            -DCMAKE_INSTALL_PREFIX=${install_dir}/moab
+make
+make install
+
+# Adding MOAB/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+add_export_var_to_bashrc 'LD_LIBRARY_PATH' "${install_dir}/moab/lib"
+add_export_var_to_bashrc 'LIBRARY_PATH' "${install_dir}/moab/lib"
+
+# Adding MOAB/include to $CPLUS_INCLUDE_PATH and $_INCLUDE_PATH
+add_export_var_to_bashrc 'CPLUS_INCLUDE_PATH' "${install_dir}/moab/include"
+add_export_var_to_bashrc 'C_INCLUDE_PATH' "${install_dir}/moab/include"
+
+# Adding pymoab to $PYTHONPATH
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version.split('')[0][0:3])')
+add_export_var_to_bashrc 'PYTHONPATH' "$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages"
+
+
+#############
+### DAGMC ###
+#############
+
+# Making sure MOAB is in the LD_LIBRARY_PATH
+source ~/.bashrc
+echo "LD_LIBRARY_PATH ${LD_LIBRARY_PATH}"
+
+install_dir=$1
+
+# pre-setup
+cd ${install_dir}
+check_repo dagmc
+mkdir -p dagmc
+cd dagmc
+
+# clone and version
+git clone https://github.com/svalinn/DAGMC.git dagmc-repo
+cd dagmc-repo
+git checkout develop
+mkdir build
+cd build
+
+# cmake, build and install
+cmake .. -DMOAB_DIR=${install_dir}/moab \
+            -DBUILD_STATIC_LIBS=OFF \
+            -DCMAKE_INSTALL_PREFIX=${install_dir}/dagmc
+make
+make install
+
+# Adding DAGMC/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+add_export_var_to_bashrc "LD_LIBRARY_PATH" "${install_dir}/dagmc/lib"
+add_export_var_to_bashrc "LIBRARY_PATH" "${install_dir}/dagmc/lib"
+
+# Adding dagmc/bin to $PATH
+add_export_var_to_bashrc "PATH" "${install_dir}/dagmc/bin"
```

Comment:
this function seems overly complex and unnecessary. Users may not realize what it is doing.

Reply:
I got the point

I was scared of putting the exposing echo statement would be overly complicated...